ELSpaceTests/TestCases/Containers/AppContainer_PresentersSpec.swift

```diff
@@ -0,0 +1,25 @@
+import Quick
+import Nimble
+
+@testable import ELSpace
+
+class AppContainer_PresentersSpec: QuickSpec {
+
+    override func spec() {
+        describe("AppContainer_Presenters") {
+            var sut: AppContainer!
+
+            beforeEach {
+                sut = AppContainer()
+            }
+
+            it("should build ViewControllerPresenter") {
+                expect(sut.viewControllerPresenter).to(beAnInstanceOf(ViewControllerPresenter.self))
+            }
+
+            it("should build ModalViewControllerPresenter") {
+                expect(sut.modalViewControllerPresenter).to(beAnInstanceOf(ModalViewControllerPresenter.self))
+            }
+        }
```

Comment:
Please also test if `basicModalPresentationConfiguration` is being built properly

Reply:
👍